### PR TITLE
Fix forked component state

### LINKED ISSUE #348: [BUG] Forked component changes do not persist in LS
**Describe the bug:**
When forking an existing component and making changes to source code the changes do not persist in local storage.

**To Reproduce:**
1. Go to near.org and fork any existing component
2. Make changes to the source code
3. Hit the "back" button on your browser 
4. Hit "forward" button on your browser
5. See changes deleted

**Expected behavior:**
Step 5 should have changes still present. 

**Additional context:**
This issue does not present itself when creating a "new" component.

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Closes #348

### CODE REVIEW

**Comment by calebjacob** [comment]
I'm not super familiar with the sandbox editor code, so I'll let @marcinbodnar be the one to approve this one. :)

**Review by marcinbodnar — CHANGES_REQUESTED** [review]
@shelegdmitriy It seems like it doesn't solve the problem, try refreshing the page.

I can take this issue from you, it's a bit problematic, and I think I was fixing the same issue a few months ago.

**Comment by shelegdmitriy** [comment]
I actually tried and it was working

**Comment by shelegdmitriy** [comment]
Have you tried to run my branch locally?

**Comment by marcinbodnar** [comment]
@shelegdmitriy I tried on the preview.

File: src/components/sandbox/Sandbox.js [hunk 1]
```diff
@@ -440,7 +424,9 @@ export const Sandbox = ({ onboarding = false }) => {
     selectFile(newPath);
   };
 
+  // possibly unused
```
Comment (shelegdmitriy) [inline src/components/sandbox/Sandbox.js:427]: It seems this section below is unused @marcinbodnar could you confirm that?

**Comment by calebjacob** [comment]
@shelegdmitriy Both preview links seem to work for me.

**Review by calebjacob — APPROVED** [review]

**Review by marcinbodnar — APPROVED** [review]
There is an issue with the red dot that indicates if there are any changes made to the component. It doesn't disappear after you undo the changes made.

We can merge it but the issue should be addressed in a separate PR cc @charleslavon

**Comment by shelegdmitriy** [comment]
@marcinbodnar good catch! That should be fixed for sure but in a separate PR because it's very minor issue

**Pull request merged**